File: Modelo/DatosSim.java
package Modelo;

public class DatosSim {

    private double dist = 0;                     // Distancia a la superficie m
    private double acel = 0;                     // aceleración m·s-2
    private double vel = 0;                      // velocidad en m· s-1
    private double impulso = 0;                  // Impulso retrocohetes en m·s-2
    private double fuel = 0;					   // Fuel restante
    private int tiempo = 0;                    // Tiempo de simulación

    public DatosSim(double dist, double acel, double vel, double impulso, double fuel, int tiempo) {
        super();
        this.dist = dist;
        this.acel = acel;
        this.vel = vel;
        this.impulso = impulso;
        this.fuel = fuel;
        this.tiempo = tiempo;
    }

    public double getDist() {
        return dist;
    }

    public void setDist(double dist) {
        this.dist = dist;
    }

    public double getAcel() {
        return acel;
    }

    public void setAcel(double acel) {
        this.acel = acel;
    }

    public double getVel() {
        return vel;
    }

    public void setVel(double vel) {
        this.vel = vel;
    }

    public double getImpulso() {
        return impulso;
    }

    public void setImpulso(double impulso) {
        this.impulso = impulso;
    }

    public double getFuel() {
        return fuel;
    }

    public void setFuel(double fuel) {
        this.fuel = fuel;
    }

    public int getTiempo() {
        return tiempo;
    }

    public void setTiempo(int tiempo) {
        this.tiempo = tiempo;
    }

    public String toString() {

        return "";
    }

}

File: Modelo/DatosSim.php
<?php

class DatosSIM
{
    // Zona de declaración de variables
    private $dist = double(0);
    private $acel = double(0);
    private $vel = double(0);
    private $impulso = double(0);
    private $fuel = double(0);
    private $tiempo = int(0);

    // Constructor
    public function __construct($dist, $acel, $vel, $impulso, $fuel, $tiempo) {
        $this->dist = $dist;
        $this->acel = $acel;
        $this->vel = $vel;
        $this->impulso = $impulso;
        $this->fuel = $fuel;
        $this->tiempo = $tiempo;
    }

    // Getters & Setters
    public function get_dist() {
        return $this->dist;
    }
    public function get_acel() {
        return $this->acel;
    }
    public function get_vel() {
        return $this->vel;
    }
    public function get_impulso() {
        return $this->impulso;
    }
    public function get_fuel() {
        return $this->fuel;
    }
    public function get_tiempo() {
        return $this->tiempo;
    }
    public function set_dist($dist) {
        $this->dist = $dist;
    }
    public function set_acel($acel) {
        $this->acel = $acel;
    }
    public function set_vel($vel) {
        $this->vel = $vel;
    }
    public function set_impulso($impulso) {
        $this->impulso = $impulso;
    }
    public function set_fuel($fuel) {
        $this->fuel = $fuel;
    }
    public function set_tiempo($tiempo) {
    $this->tiempo = $tiempo;
    }

    public function __tostring() {
        return "";
    }

}
